tools/topology/topology1/sof/pipe-gui-components-capture.m4

```diff
@@ -0,0 +1,146 @@
+# Capture EQ Pipeline and PCM, 48 kHz
+#
+# Pipeline Endpoints for connection are :-
+#
+#  host PCM_C <-- B0 <-- Volume 0 <-- B1 <--EQ_IIR 0 <-- B2 <-- sink DAI0
+
+# Include topology builder
+include(`utils.m4')
+include(`buffer.m4')
+include(`pcm.m4')
+include(`pga.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`bytecontrol.m4')
+include(`mixercontrol.m4')
+include(`eq_iir.m4')
```

Comment:
Instead of only eq_iir, can we add also eq_fir?
Or actually use this topology: [pipe-eq-iir-eq-fir-volume-playback.m4](https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/sof/pipe-eq-iir-eq-fir-volume-playback.m4)